includes/api/class-wc-admin-rest-reports-customers-controller.php

```diff
@@ -60,7 +61,7 @@ protected function prepare_reports_query( $request ) {
 		$args['avg_order_value_max'] = $request['avg_order_value_max'];
 		$args['last_order_before']   = $request['last_order_before'];
 		$args['last_order_after']    = $request['last_order_after'];
-		$args['customers']           = $request['customers'];
+		$args['include']             = $request['include'];
```

Comment:
Remove the `customers` parameter breaks the table search on the customers report.

Reply:
Thanks for pointing that out, I tried to see where this was used but didn't see anything when I looked.

Instead of updating this to `include` here, I've only updated the parameter on the main `/wc/v4/customers` endpoint, which matches other endpoints (i.e. /wc/v4/coupons?include is used for the main endpoint and `/wc/v4/reports/coupons/?coupons=... is used for filtering)

Reply:
I think this is where it's used: https://github.com/woocommerce/wc-admin/blob/463200547c242bf38131b234fe5b9713b6a4268a/client/wc-api/reports/utils.js#L42